klient/k8s/resources/resources.go

```diff
@@ -189,6 +189,11 @@ func (r *Resources) GetScheme() *runtime.Scheme {
 	return r.scheme
 }
 
+// GetClient return the controller-runtime client instance
+func (r *Resources) GetClient() cr.Client {
```

Comment:
Because there is function `klient.New()` that returns a `klient.Client`. Maybe this should be named to reflect what it is returning i.e., `GetRawClient` or `GetCRClient` or `GetControllerRuntimeClient`